src/main/resources/avro/rnaquantification.avdl

```diff
@@ -0,0 +1,216 @@
+@namespace("org.ga4gh.models")
+/**
+This protocol defines feature expression counts on GA4GH reads. 
+*/
+protocol RnaQuantifications {
+
+import idl "common.avdl";
+import idl "reads.avdl";
+
+/**
+Units for expression level
+*/
+enum ExpressionUnits {
+  FPKM,
+  RPM
+}
+
+/**
+Top level identifying information
+*/
+record RnaQuantification {
+  /** 
+  The unique ID assigned to the results of running the described programs on the
+specified reads and assignment to the listed annotation.
+  */
+  string id;
+ 
+  /**
+  Name
+  */
+  union {null, string} name = null;
```

Comment:
id and name should follow the standards defined here:
http://ga4gh-schemas.readthedocs.org/en/master/api/apidesign_intro.html

Maciek Smuga-Otto <notifications@github.com> writes:
> In src/main/resources/avro/rnaquantification.avdl:
> 
> > +}
> > +
> > +/**
> > +Top level identifying information
> > +*/
> > +record RnaQuantification {
> > +  /**
> > +  The unique ID assigned to the results of running the described programs on the
> > +specified reads and assignment to the listed annotation.
> > +  */
> > +  string id;
> > +
> > +  /**
> > +  Name
> > +  */
> > +  union {null, string} name = null;
> 
> What's the intended use of the name field? Does it correspond to a common
> naming scheme for quantifications used in public expression databases or
> somesuch?
> 
> —
> Reply to this email directly or view it on GitHub.*
>